websites/R/RTL+/presence.ts

```diff
@@ -0,0 +1,142 @@
+const presence = new Presence({
+		clientId: "1033504954763198545",
+	}),
+	getStrings = async () => {
+		return presence.getStrings(
+			{
+				play: "general.playing",
+				pause: "general.paused",
+			},
+			await presence.getSetting<string>("lang").catch(() => "en")
+		);
+	},
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video: HTMLVideoElement,
+	vidTitle: string,
+	strings: Awaited<ReturnType<typeof getStrings>> = null;
+
+presence.on("UpdateData", async () => {
+	const [startTimestamp, endTimestamp] = presence.getTimestamps(
+			Math.floor(video?.currentTime),
+			Math.floor(video?.duration)
+		),
+		presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/fMMsZfV.png",
+		},
+		{ pathname, href } = document.location;
+
+	if (!strings) strings = await getStrings();
+
+	if (document.querySelector("#bitmovinplayer-video-player_container")) {
+		video = document.querySelector(
+			"#bitmovinplayer-video-player_container"
+		) as HTMLVideoElement;
+
+		if (!isNaN(video.duration)) {
+			const footerEles = document.querySelectorAll("li.ng-star-inserted"),
+				cover: HTMLImageElement = document.querySelector("img.dvd-cover");
+
+			presenceData.smallImageKey = video.paused
+				? "https://i.imgur.com/C6mbMYz.png"
+				: "https://i.imgur.com/crCKEaC.png";
+			presenceData.smallImageText = video.paused ? strings.pause : strings.play;
+			presenceData.startTimestamp = startTimestamp;
+			presenceData.endTimestamp = endTimestamp;
+			presenceData.buttons = [
+				{
+					label: "View",
+					url: href,
+				},
+			];
+
+			if (pathname.includes("/serien/") || pathname.includes("/shows/")) {
+				const season = footerEles[3]
+						?.querySelector("a>span")
+						?.textContent.replace("Staffel ", "")
+						.trim(),
+					episode = footerEles[4]
+						?.querySelector("span")
+						?.textContent.split(" Folge ");
+
+				vidTitle = footerEles[2]?.querySelector("a>span")?.textContent.trim();
+
+				presenceData.state = `S${season}:E${episode?.[1]?.trim()} ${episode?.[0]?.trim()}`;
+				presenceData.buttons[0].label = "Watch Series";
+			} else if (pathname.includes("/filme/")) {
+				vidTitle = footerEles[2]?.querySelector("a>span")?.textContent.trim();
+				presenceData.buttons[0].label = "Watch Movie";
+			}
+
+			if (!vidTitle) {
+				vidTitle = document.title
+					.replace("im Online Stream | RTL+", "")
+					.replace("im Online Stream ansehen | RTL+", "")
+					.trim();
+			}
+
+			if (cover) presenceData.largeImageKey = cover.src;
+
+			presenceData.details = vidTitle;
+
+			if (video.duration <= 5) {
+				delete presenceData.startTimestamp;
+				delete presenceData.endTimestamp;
+
+				presenceData.smallImageKey = "https://i.imgur.com/C6mbMYz.png";
```

Comment:
Used multiple times so you can use an `Assets` enum, shown here
https://github.com/PreMiD/Presences/blob/01054ec857bc5052c45a0ee39a044ce21dd0bf85/websites/M/MangaFreak/presence.ts#L4-L8